### PR TITLE
Initial Implementation of Sentry.

### PR DESCRIPTION
**Pull request by vishwesh3**
Removed fabric from Android.

Removed fabtic from ios

### CODE REVIEW

File: android/app/src/main/java/com/zulipmobile/MainActivity.java [hunk 1]
```diff
@@ -1,8 +1,6 @@
 package com.zulipmobile;
 
 import com.facebook.react.ReactActivity;
-import com.smixx.fabric.FabricPackage;
-import com.oblador.vectoricons.VectorIconsPackage;
```
Comment (borisyankov) [inline android/app/src/main/java/com/zulipmobile/MainActivity.java]: Why remove `VectorIconsPackage`?

File: android/sentry.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+defaults.url=zulip/zulip-mobile
+defaults.org=zulip
+defaults.project=zulip-mobile
+auth.token=getTokenFrom https://sentry.io/api/
```
Comment (borisyankov) [inline android/sentry.properties]: Is there a way to read that from external file? Or will we need to replace manually before build? Maybe replace with something easier to search for like SENTRY-API-TOKEN-HERE

File: package.json [hunk 3]
```diff
@@ -1,114 +1,113 @@
...
-    "configurations": {
-      "ios.sim.debug": {
-        "binaryPath": "ios/build/Build/Products/Debug-iphonesimulator/ZulipMobile.app",
-        "build": "xcodebuild -project ios/ZulipMobile.xcodeproj -scheme ZulipMobile -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build",
-        "type": "ios.simulator",
-        "name": "iPhone 6"
-      }
-    }
-  }
+	"name": "ZulipMobile",
```
Comment (borisyankov) [inline package.json]: Do not reformat this file.

File: android/app/src/main/java/com/zulipmobile/MainActivity.java (hunk 1 above)
Comment (vishwesh3) [inline android/app/src/main/java/com/zulipmobile/MainActivity.java]: There is only import no usage. https://github.com/zulip/zulip-mobile/blob/master/android/app/src/main/java/com/zulipmobile/MainActivity.java (so may be automatically removed by Android Studio)

File: android/sentry.properties (hunk 2 above)
Comment (vishwesh3) [inline android/sentry.properties]: For now replaced it with SENTRY-API-TOKEN-HERE. We need to replace this with API before every release.

**Comment by vishwesh3** [comment]
@borisyankov reverted back all the auto formated changes 👍 (in package.json, android manifest and iOS info.plist)

**Comment by borisyankov** [comment]
I know it is super-annoying, so thanks for the effort 👍

**Pull request merged**